src/other/utils.jl

```diff
@@ -83,3 +83,10 @@ else
 end
 
 funname(c::ComposedFunction) = Symbol(funname(c.outer), :_, funname(c.inner))
+
+"""
+## Dict(df::AbstractDataFrame)
+Converts an AbstractDataFrame into JSON-friendly 1-dimensional key data.
+
+"""
+Dict(df::DataFrame) = Dict([name => Array(col) for (name,col) in zip(names(df), eachcol(df))])
```

Comment:
This is the way to define it:
```suggestion
Base.Dict(df::AbstractDataFrame) = Dict(pairs(eachcol(df)))
```